src/main/java/org/broadinstitute/hellbender/tools/funcotator/FuncotatorDataSourceDownloader.java

```diff
@@ -71,17 +71,17 @@
     //==================================================================================================================
     // Private Static Members:
 
-    private static String BASE_URL = "gs://broad-public-datasets/funcotator/funcotator_dataSources.v1.6.20190124";
+    private static String BASE_URL = "gs://broad-public-datasets/funcotator/funcotator_dataSources.v1.7.20200521";
 
     private static String GERMLINE_GCLOUD_DATASOURCES_BASEURL     = BASE_URL + "g";
     @VisibleForTesting
     static Path   GERMLINE_GCLOUD_DATASOURCES_PATH        = IOUtils.getPath(GERMLINE_GCLOUD_DATASOURCES_BASEURL + ".tar.gz");
     @VisibleForTesting
     static Path   GERMLINE_GCLOUD_DATASOURCES_SHA256_PATH = IOUtils.getPath(GERMLINE_GCLOUD_DATASOURCES_BASEURL + ".sha256");
 
-    private static String SOMATIC_GCLOUD_DATASOURCES_BASEURL     = BASE_URL + "s";
-    @VisibleForTesting
-    static Path   SOMATIC_GCLOUD_DATASOURCES_PATH        = IOUtils.getPath(SOMATIC_GCLOUD_DATASOURCES_BASEURL + ".tar.gz");
+    public static String SOMATIC_GCLOUD_DATASOURCES_BASEURL     = BASE_URL + "s";
+
+    public static Path   SOMATIC_GCLOUD_DATASOURCES_PATH        = IOUtils.getPath(SOMATIC_GCLOUD_DATASOURCES_BASEURL + ".tar.gz");
```

Comment:
Same question: can these be final and package access instead of public?

Reply:
A lot of fields here can be made `final`, but the same package issue exists here as in the above case.